comms/discovery/rpcz/apply.go

```diff
@@ -107,7 +107,7 @@ func (proc *RPCProcessor) ApplyAndPublish(rpcLog *schema.RpcLog) (*schema.RpcLog
 	// publish event
 	j, err := json.Marshal(rpcLog)
 	if err != nil {
-		slog.Error("err: invalid json", err)
+		slog.Error("err: invalid json", "err", err)
```

Comment:
Yeah... really it should never hit this invalid JSON condition (since this would be caught in the `mutate` endpoint) unless you called this function directly via a test or via a CLI command with invalid input.

I suppose it could be a panic